pkgs/tools/filesystems/grive2/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, pkgconfig, fetchurl, yajl, cmake, libgcrypt, curl, expat, boost, binutils }:
+
+stdenv.mkDerivation rec {
+  version = "0.5.0";
+  name = "grive2-${version}";
+
+  src = fetchFromGitHub {
+    owner = "vitalif";
+    repo = "grive2";
+    rev =  "ae06eccb38b2fe250c9ddeac3e3973f80b8a0aa9";
+    sha256 = "0gyic9228j25l5x8qj9xxxp2cgbw6y4skxqx0xrq6qilhv4lj23c";
+  };
+
+  nativeBuildInputs = [cmake pkgconfig];
```

Comment:
The usual style is to leave a space after/before brackets, as in `[ a b c d e ]`.